couchbase/datadog_checks/couchbase/data/auto_conf.yaml

```diff
@@ -1,21 +1,30 @@
+## @param ad_identifiers - list of strings - required
+## A list of container identifiers that are used by Autodiscovery to identify
+## which container the check should be run against. For more information, see:
+## https://docs.datadoghq.com/agent/guide/ad_identifiers/
+#
 ad_identifiers:
   - couchbase
 
+## All options defined here are available to all instances.
+#
 init_config:
 
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param server - string - required
     ## The server's url.
     #
   - server: http://%%host%%:8091
 
-    ## @param username - string - optional
+    ## @param user - string - optional
     ## The username to use if `server` or `query_monitoring_url` is behind basic auth.
     #
-    user: Administrator
+    # user: Administrator
 
     ## @param password - string - optional
     ## The password to use if `server` or `query_monitoring_url` is behind basic auth.
     #
-    password: password
+    # password: password
```

Comment:
I think there is no need to have `user` and `password`. The section can be removed IMO. 

This `auto_conf.yaml` only works when there is no auth.

Reply:
At first I thought that `Administrator` and `password` were the default credentials in couchbase, but it seems they are not https://docs.couchbase.com/server/current/cli/cbcli/couchbase-cli-cluster-init.html. So I guess we can remove `password` and `user` from the auto_conf